pkg/adapter/adapter.go

```diff
@@ -153,20 +154,18 @@ func (a *mongoDbAdapter) makeCloudEvent(data bson.M) (*cloudevents.Event, error)
 
 	// Set cloud event specs and attributes. TODO: issue #43
 	// 		ID     -> id of mongo change object
-	// 		Source -> database/collection.
+	// 		Source -> Source environment variable.
 	// 		Type   -> type of change either insert, delete or update.
 	//		Data   -> data payload containing either id only for
 	//                deletion or full object for other changes.
-	event.SetID(data["_id"].(bson.M)["_data"].(string))
+	change, err := utils.DecodeChangeBson(data)
```

Comment:
same here, let's just have a docs PR without this changes and you can include them later